manifests/x/Xidicone/ZeusLite/3.97z/Xidicone.ZeusLite.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Xidicone.ZeusLite
+PackageVersion: 3.97z
+InstallerSwitches:
+  Silent: /VERYSILENT
+  SilentWithProgress: /SILENT

```

This should do it. It will automatically choose user install (this is recommended)
```suggestion
  SilentWithProgress: /SILENT
  Custom: /CURRENTUSER
```